install.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash -e
```

Comment:
What's wrong with -e?

Reply:
I couldn't get it to work, I also don't see it on the manpage for bash. What is it supposed to do?

Reply:
Stop the script if there's an error.

Reply:
Oh, there's a portable way to do that, give me one moment.

Reply:
Done :)